src/styles/main.css

```diff
@@ -1,3 +1,23 @@
+@import "https://unpkg.com/open-props";
```

Comment:
not sure if something is getting confused here, but open-props are already being defined in _theme.css_ as well some of these global styles.